build.sbt

```diff
@@ -10,7 +11,8 @@ lazy val root = project
   .aggregate(core)
   .settings(
     name := "play-ebean-root",
-    releaseCrossBuild := false
+    releaseCrossBuild := false,
```

Comment:
This setting is documented as "whether the release should be cross built", which to me suggests we should set it to `true`.